esp-hal/build.rs

```diff
@@ -128,11 +128,18 @@ fn main() -> Result<(), Box<dyn Error>> {
     // emit config
     generate_config(
         "esp_hal",
-        &[(
-            "place-spi-driver-in-ram",
-            Value::Bool(false),
-            "Places the SPI driver in RAM for better performance",
-        )],
+        &[
+            (
+                "place-spi-driver-in-ram",
+                Value::Bool(false),
+                "Places the SPI driver in RAM for better performance",
+            ),
+            (
+                "spi-address-workaround",
```

Comment:
Do you reckon libraries may want to set this?

Reply:
There should be one source of truth for the config, and it should lie in the hands of the user. In this case, we've enabled it by default, which is a sane thing to do, and users can opt-out. One thing libraries can do is detect if the setting is enabled and either warn/panic if it should be - but even this I would discourage unless the library functionality depended on it.